### PR TITLE
Update QQ.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/QQ.xml [hunk 1]
```diff
@@ -281,7 +278,12 @@
 	<target host="i.mail.qq.com" />
 	<target host="res.mail.qq.com" />
 	<target host="rl.mail.qq.com" />
+	<target host="map.qq.com" />
+		<exclusion pattern="^http://map\.qq\.com/(?!api/)" />
+		<test url="http://map.qq.com/api/js" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/QQ.xml]: `http://map.qq.com/api/js` seems to request hard-coded http urls, and https-everywhere won't have chance to rewrite the requests before they are blocked 	except Tor browser...

Comment (renyouguo) [inline src/chrome/content/rules/QQ.xml]: They will be rewrited in http site, for example, http://map.qq.com/

Comment (gloomy-ghost) [inline src/chrome/content/rules/QQ.xml]: I couldn't reproduce the rewriting, but I just realized that it is not a problem for http site importing a https script which loads another http script...

BTW, it is excluded in 93f4a12

Comment (renyouguo) [inline src/chrome/content/rules/QQ.xml]: So stupid I am.
Thank you.

Comment (renyouguo) [inline src/chrome/content/rules/QQ.xml]: @gloomy-ghost 发现了#6921
我们这两个无论先合并了谁的，另外一个都会发生冲突，要重新提交。我看你那整理得很不错，要不你把我这边的有用的部分合并过去吧，map这个纯属无聊，3gimg倒是真的，还有部分的注释修改。

Comment (gloomy-ghost) [inline src/chrome/content/rules/QQ.xml]: 没问题，明天有空的时候合并。

是你发PR到我那还是我直接从你这复制？

Comment (renyouguo) [inline src/chrome/content/rules/QQ.xml]: 你复制吧。QQ这我的手气特别差，动不动就触发自动测试停止的bug……

Comment (renyouguo) [inline src/chrome/content/rules/QQ.xml]: 欢迎围观我手气最差而最终促使官方对自动测试出错后再试一次的：#5626
所以你的360还可以继续战斗……

Comment (gloomy-ghost) [inline src/chrome/content/rules/QQ.xml]: 你这手气不做测试真是可惜了……

**Comment by renyouguo** [comment]
#6921

**Pull request closed without merging**